pkg/users/users.js

```diff
@@ -26,7 +26,7 @@ import { createRoot } from 'react-dom/client';
 import { superuser } from "superuser";
 
 import { usePageLocation, useLoggedInUser, useFile, useInit } from "hooks.js";
-import { etc_passwd_syntax, etc_group_syntax, etc_shells_syntax } from "./parsers.js";
+import { etc_passwd_syntax, etc_group_syntax, etc_shells_syntax } from "parsers.js";
```

Comment:
Please rename the file then -- "parsers.js" is too generic as a public API. Perhaps `nss_parser` or `pam_user_parser.js` ?

Reply:
@martinpitt done :)